media-sound/pms/pms-9999.ebuild

```diff
@@ -39,17 +39,12 @@ pkg_pretend() {
 }
 
 src_prepare() {
-	eapply "${FILESDIR}/pms-9999-fix-automagic-dep.patch"
-	eapply_user
+	default
 
 	eautoreconf
```

Comment:
do you still need eautoreconf after removing the buildsystem patch or could you just drop the whole src_prepare?